packages/components/Callout/src/Callout.types.ts

```diff
@@ -97,8 +98,9 @@ export interface ICalloutTokens extends IBackgroundColorTokens, CalloutBorderTok
   minPadding?: number;
 }
 
-export interface ICalloutProps extends ViewProps, ICalloutTokens {
-  /*
+export interface ICalloutProps extends IViewProps, ICalloutTokens {
+  /**
+   * @platform win32
```

Comment:
@rurikoaraki Something I've been meaning to do is add these tags for platform specific props. Just an FYI